asgi.py

```diff
@@ -13,7 +13,7 @@
             host=settings.host,
             port=settings.port,
             reload=settings.app_reload,
-            log_config=LOGGING_CONFIG,
+            log_config=None,
```

Comment:
I didn't actually verify this, but I assumed that if we managed logging through unicorn like this, we wouldn't actually set filter on the `console` handler to make assertions on in tests